WIP-CHANGELOG.md

```diff
@@ -29,3 +32,6 @@
 - Update Trivy Operator Dashboard to improve the user experience
 - Bump sops to version `3.7.3`.
 - Update Falco rules
+- Updated harbor chart to `v1.12.1` which upgraded harbor to `v2.8.1`
```

Comment:
They have now released chart version 1.12.2 with harbor version 2.8.2. So if you want you could try to upgrade to that, seemed like there were only some small changes.

Reply:
Right, forgot about that. Will do that :+1:

Reply:
Done